src/evaluate/load.py

```diff
@@ -600,34 +617,35 @@ def _get_modification_time(module_hash):
             f"couldn't be found locally at {self.name}, or remotely on the Hugging Face Hub."
         )
         # make the new module to be noticed by the import system
-        module_path = ".".join([os.path.basename(dynamic_modules_path), "metrics", self.name, hash, self.name])
+        module_path = ".".join([os.path.basename(dynamic_modules_path), module_namespace, self.name, hash, self.name])
         importlib.invalidate_caches()
         return MetricModule(module_path, hash)
 
 
-def metric_module_factory(
+def evaluate_module_factory(
```

Comment:
(nit) I like evaluation_module_factory a bit better, but feel free to ignore if you think it's not ideal

```suggestion
def evaluation_module_factory(
```